internal/servers/server/option.go

```diff
@@ -183,7 +189,12 @@ func WithListenConfig(lc *net.ListenConfig) Option {
 
 func WithServerMode(m mode) Option {
 	return func(s *server) {
-		s.mode = m
+		switch m {
+		case GRPC, REST, GQL:
+			s.mode = m
+		default:
```

Comment:
we don't need return and default statement

Reply:
Thank you for your feedback.
I fixed about this contents.